src/hooks/useAnimation.ts

```diff
@@ -6,24 +6,26 @@ interface AnimationState {
   result: string | null;
 }
 
+interface UseAnimation {
+  state: AnimationState;
+  resultSpin: () => void;
+  startIndefiniteSpin: () => void;
+  stopIndefiniteSpin: () => void;
+  scope: React.RefObject<HTMLDivElement>;
+}
```

Comment:
RE: Support an indefinitely spinning wheel PRIOR to confirming a MM tx window 

Spinning the wheel indefinitely until the tx is confirmed by user. Once the user has confirmed the tx, we can then invoke our `resultSpin()` which is a timed wheel spinner that IS NOT indefinite and would conclude with a result.

Reply:
Merging in this approach.